src/services/token.service.js

```diff
@@ -0,0 +1,29 @@
+const { Token } = require('../models/Token.model');
+
+const save = async (userId, newToken) => {
+  const token = await Token.findOne({ where: { UserId: userId } });
+
+  if (!token) {
+    await Token.create({ UserId: userId, refreshToken: newToken });
+
+    return;
+  }
+
+  token.refreshToken = newToken;
+
+  await token.save();
+};
+
+const getByToken = async (refreshToken) => {
+  return Token.findOne({ where: { refreshToken } });
```

Comment:
Similar to the previous comment, consider adding error handling for the getByToken function to manage any issues that might arise during the database query.